x/ibc/02-client/client/cli/tx.go

```diff
@@ -118,36 +118,33 @@ $ %s tx ibc client update [client-id] [path/to/header.json] --from node0 --home
 // https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#misbehaviour
 func GetCmdSubmitMisbehaviour(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "misbehaviour [client-it] [path/to/evidence.json]",
+		Use:   "misbehaviour [path/to/evidence.json]",
```

Comment:
Don't we need the `client-id` to submit the evidence, or does the evidence contain the `client-id`?

Reply:
the evidence contains `client-id`